### PR TITLE
Add Bower support

### PR DESCRIPTION
**Pull request by demiankatz**
This PR adds a bower.json file as suggested in issue #9.

Once this is merged, the project needs to be registered with Bower as described here:

https://bower.io/docs/creating-packages/#register

(Just a matter of installing Bower and running `bower register leaflet-color-markers https://github.com/pointhi/leaflet-color-markers`).

Bower also expects tagged versions using semantic versioning, so it would be great to tag the current master branch with v1.0.0 to get things started.

Please let me know if you have questions or concerns or if I can do anything else to help out. (Admittedly, I'm new to Bower as well, so apologies if I've made any newbie mistakes... but it seems pretty straightforward).

Thanks again!

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  "name": "leaflet-color-markers",
+  "homepage": "https://github.com/pointhi/leaflet-color-markers",
+  "authors": [
+    "Thomas Pointhuber <thomas.pointhuber@gmx.at>"
+  ],
+  "description": "color variations of the standard leaflet marker",
+  "main": "js/leaflet-color-markers.js",
+  "license": "BSD-2-Clause",
+  "dependencies": {
+    "leaflet": ">=0.7.0"
```
Comment (pointhi) [inline bower.json:11]: technically, this repo has no dependencies

Comment (demiankatz) [reply bower.json:11]: @pointhi, the Javascript file in the repo uses Leaflet code (specifically, a dependency on L.Icon), which is why I put this here... but of course I understand the icons can potentially be useful in other contexts as well. I wouldn't object to dropping this section, or changing "dependencies" to "devDependencies" to show that the relationship is optional (though that's not exactly the right semantic use of devDependencies).

**Comment by pointhi** [comment]
thanks, I will try to register it with bower now

**Comment by demiankatz** [comment]
Thanks, @pointhi, it is working for me! I appreciate your help.

**Pull request merged**